src/test/java/org/broadinstitute/dsm/route/KitUploadRouteTest.java

```diff
@@ -85,7 +81,7 @@ public void fileUserNameDoesNotMatch() {
 
         ParticipantWrapper testParticipant = participantFactory("Mickey", "Mouse", "");
 
-        Assert.assertFalse(route.isKitUploadNameMatchesToEsName(participantFirstNameFromDoc, participantLastNameFromDoc, Optional.of(testParticipant)));
+        Assert.assertNotNull(route.isKitUploadNameMatchesToEsName(participantFirstNameFromDoc, participantLastNameFromDoc, Optional.of(testParticipant)));
```

Comment:
Can it be null? I'm asking because message in isKitUploadNameMatchesToEsName is initialized as empty string

Reply:
Good catch! Thanks!